### PR TITLE
Show summary of changes for deploy and download commands

### PR DESCRIPTION
**Pull request by andyw8**
### Warn Checklist
- [ ] This changes the interface and requires a Major/Minor version change.
- [x] I have :tophat:'d these changes by using the commands I changed by hand.
- [ ] I have added a dependancy to the project.

### CODE REVIEW

File: cmd/deploy.go [hunk 1]
```diff
@@ -72,9 +76,19 @@ func deploy(ctx *cmdutil.Ctx) error {
...
+			updateCount++
+		case file.Skip:
+			skipCount++
+		case file.Remove:
+			removeCount++
+		}
 	}
 
 	deployGroup.Wait()
+	fmt.Printf("Updated: %d, Removed: %d, Skipped: %d", updateCount, removeCount, skipCount)
```
Comment (andyw8) [inline cmd/deploy.go:91]: This works, except that the final line of output gets shown afterwards, e.g.

```
[development] Skipped templates/customers/account.liquid (Checksum: 4fe35d7a307346edefa4b53d554ab06b)
[development] Skipped templates/article.liquid (Checksum: 20defecae451efdebc626d58cee1a87f)
Updated 0, Removed 0, skipped 25
[development] Skipped config/settings_data.json (Checksum: e8175e268a84d3861e3074569e0f986c)
```
Any idea why that is @tanema ?

**Comment by andyw8** [comment]
@tanema this appears to work, but I'm getting some `DATA RACE` warnings in the tests, e.g.:

```
==================
WARNING: DATA RACE
Read at 0x00c0000247c8 by goroutine 41:
  github.com/Shopify/themekit/cmd.download.func1()
      /Users/andyw8/src/github.com/Shopify/themekit/cmd/download.go:60 +0x799

Previous write at 0x00c0000247c8 by goroutine 39:
  github.com/Shopify/themekit/cmd.download.func1()
      /Users/andyw8/src/github.com/Shopify/themekit/cmd/download.go:60 +0x7b5

Goroutine 41 (running) created at:
  github.com/Shopify/themekit/cmd.download()
      /Users/andyw8/src/github.com/Shopify/themekit/cmd/download.go:46 +0x337
  github.com/Shopify/themekit/cmd.TestDownload()
      /Users/andyw8/src/github.com/Shopify/themekit/cmd/download_test.go:28 +0x437
  testing.tRunner()
      /Users/andyw8/.dev/go/1.12.9/src/testing/testing.go:865 +0x163

Goroutine 39 (finished) created at:
  github.com/Shopify/themekit/cmd.download()
      /Users/andyw8/src/github.com/Shopify/themekit/cmd/download.go:46 +0x337
  github.com/Shopify/themekit/cmd.TestDownload()
      /Users/andyw8/src/github.com/Shopify/themekit/cmd/download_test.go:28 +0x437
  testing.tRunner()
      /Users/andyw8/.dev/go/1.12.9/src/testing/testing.go:865 +0x163
==================
```
I haven't dug in deep yet, just pinging in case there's something obvious that I've missed.

**Comment by andyw8** [comment]
@tanema seems I'll need to use atomic functions for handling the incrementing, so looking into that.

**Comment by andyw8** [comment]
cc @Nupur-Mukherjee for any thoughts on this suggestions.

**Comment by Nupur-Mukherjee** [comment]
@andyw8 I agree with @tanema 
Lets use "No change in" in place of "skipped" for more clarity.

**Pull request merged**